src/locate/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Comments in Dockerfiles
+FROM python:3.7
```

Comment:
We should not be having file changes inside another microservice which this PR does not address. Noticed changes inside `src/locate` 🤔